Documentation/HowToUseReleaseDrafter.md

```diff
@@ -0,0 +1,69 @@
+# GitHub release configuration
```

Comment:
Thinking if we can rename this file to something like "Documentation/ReleaseProcess.md", where the Release drafter configuration is one section of it.